lib/getstats.js

```diff
@@ -74,8 +75,11 @@ function getActiveIceCandidatePairStats(peerConnection, options) {
   options = options || {};
 
   if (typeof options.testForChrome !== 'undefined'
-    || typeof webkitRTCPeerConnection !== 'undefined') {
-    return peerConnection.getStats().then(standardizeChromeActiveIceCandidatePairStats);
+    || typeof webkitRTCPeerConnection !== 'undefined'
+    || typeof options.testForSafari  !== 'undefined'
+    || guessBrowser() === 'safari') {
```

Comment:
@syerrapragada Since you've used `guessBrowser()` here, can you replace all prefixed RTCPeerConnection references with this?